SU2_PY/SU2/eval/gradients.py

```diff
@@ -467,7 +467,10 @@ def findiff( config, state=None ):
         log_findiff = None
 
     # evaluate step
-    step = 0.001 * float(config.REF_LENGTH_MOMENT)
+    if config.has_key('FIN_DIFF_STEP'):
+        step = float(config.FIN_DIFF_STEP)
+    else:
+        step = 0.001 * float(config.REF_LENGTH_MOMENT)
```

Comment:
Ok, will do (both on removing the REF_LENGTH_MOMENT option and making sure FIN_DIFF_STEP is in the config file)